### ISSUE TITLE
7.12 TCPDF ? instead of local characters

### ISSUE THREAD

**Issue by maarisk** [salesagility/SuiteCRM#9350]
After upgrade to SuiteCRM 7.12 and with TCPDF engine enabled, PDFs generated contain question mark instead of specific local characters.

#### Issue
If PDF template contains language specific characters (eg. latvian characters ķ, ē, ģ, ņ) in the resulting PDF generated with TCPDF engine, these are replaced with question marks (?).

#### Expected Behavior
Generated PDF represents all local specific characters correctly as on template.

#### Actual Behavior
Local characters in most cases are replaced with ?.

#### Steps to Reproduce

1. Upgrade to SuiteCRM 7.12
2. Ensure that TCPDF engine is enabled under PDF settings
3. Create Invoice PDF template containing characters ķ, ģ, ū, ē, ņ, ā, č, ļ. (eg. Rēķins)
4. Print invoice as PDF using created template.

#### Context
Invoice PDF is generated from SuiteCRM and sent to customers. For local customers invoices are in local langiage (Latvian) and contain words where local characters are present. Sending of invoices containing ? instead of relevant loca chars is not acceptable.

#### Your Environment

* SuiteCRM Version used: 7.12
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Safari 15.1
* Environment name and version (e.g. MySQL, PHP 7): MySQL 8, PHP 7.4.23
* Operating System and version (e.g Ubuntu 16.04): macOS 11.6

**Comment by tsmgeek** [comment]
Which fonts are you using?
What language are you writing in?

**Comment by maarisk** [comment]
See comments below.

Māris



> On 7 Nov 2021, at 21:04, Ricardo ***@***.***> wrote:
> 
> 
> Which fonts are you using?
> 
Font used is Helvetica Neu.
> What language are you writing in?
> 
Language used in template is Latvian.

**Comment by tsmgeek** [comment]
https://stackoverflow.com/questions/5333702/tcpdf-utf-8-symbols-not-showing-up

Might be issue with font not supporting those utf8 characters.

**Comment by billygoat-trump** [comment]
@maarisk, yes, my CRM Similar to this error, the font has no effect, the characters are almost "wrong" compared to the 7.11.20.
Besides, I also broken the framework of the table, the table seems to be limited a lot when exporting PDF.
[Situation Solution] I was forced to "use" the AOS_PDF_TEMPLATES module of version 7.11.20 for the current job (7.12), though the cache error message when executing the "Repairs and Rebuilds DB, Extensions, Vardefs, Suitecrm dashlets etc. " In the Admin Tool

**Comment by timo-ecm2** [comment]
Hi I have also a issue. After upgrade to SuiteCRM 7.12.x and with TCPDF engine enabled, PDFs generated tables with code 
<table style="width: 100%; border: 0pt none; border-spacing: 0pt;">

Now show up like below

![Naamloos](https://user-images.githubusercontent.com/51777898/189136852-bd7044d4-791f-4c7d-ad75-b868ccfaca5a.png)
